canonical-web-and-design/vanilla-framework
# Top navigation updates

## Done

This is a feature branch. Code was reviewed already in separate PR.

This just need final round of QA before merging and releasing.

## QA

- Open [demo](https://vanilla-framework-4403.demos.haus/)
- Check Vanilla docs top nav, on desktop and small screens, make sure it works as expected including search
- Review updated documentation:
  - Make sure What's new page is up to date and correct: https://vanilla-framework-4403.demos.haus/docs/whats-new
  - Review updated top nav docs: https://vanilla-framework-4403.demos.haus/docs/patterns/navigation
  - Review navigation examples:
    - https://vanilla-framework-4403.demos.haus/docs/examples#:~:text=Navigation%20/%20Default%20dark,Navigation%20/%20Search%20light

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
